<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sortowanie</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Documents and Settings/Michał/Moje dokumenty/Moje obrazy/projekt/Dummies-guy.jpg</normaloff>C:/Documents and Settings/Michał/Moje dokumenty/Moje obrazy/projekt/Dummies-guy.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0" columnstretch="0">
    <item row="1" column="0">
     <widget class="SortWidget" name="widget" native="true"/>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,2,2,2,0,1,0,1">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>SelectionSort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>InsertionSort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BubbleSort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HeapSort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MergeSort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>QuickSort</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BogoSort</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Pauza</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>n :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Opóźnienie (ms): </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,0,3,6,3,6,10">
      <item>
       <widget class="QLabel" name="labelAlgorithm">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonKod">
        <property name="text">
         <string>Kod</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Porównań:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelComparisions">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dostępów:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAccesses">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SortWidget</class>
   <extends>QWidget</extends>
   <header>sortwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
